src/vs/workbench/browser/actions/layoutActions.ts

```diff
@@ -470,49 +470,87 @@ export class ToggleStatusbarVisibilityAction extends Action2 {
 
 registerAction2(ToggleStatusbarVisibilityAction);
 
-// --- Bse Class Toggle Boolean Setting Action
+// --- Hide Editor Tabs
 
-abstract class BaseToggleBooleanSettingAction extends Action2 {
+export class HideEditorTabsAction extends Action2 {
 
-	protected abstract get settingId(): string;
+	static readonly ID = 'workbench.action.hideEditorTabs';
+
+	constructor() {
+		super({
+			id: HideEditorTabsAction.ID,
+			title: {
+				value: localize('hideEditorTabs', "Hide Editor Tabs"),
+				original: 'Hide Editor Tabs'
+			},
+			category: Categories.View,
+			precondition: ContextKeyExpr.equals('config.workbench.editor.showTabs', 'none').negate(),
+			f1: true
+		});
+	}
 
 	override run(accessor: ServicesAccessor): Promise<void> {
 		const configurationService = accessor.get(IConfigurationService);
+		return configurationService.updateValue('workbench.editor.showTabs', 'none');
+	}
+}
+registerAction2(HideEditorTabsAction);
 
-		const oldettingValue = configurationService.getValue<string>(this.settingId);
-		const newSettingValue = !oldettingValue;
+// --- Show Multiple Editor Tabs
+
+export class ShowMultipleEditorTabsAction extends Action2 {
+
+	static readonly ID = 'workbench.action.showMultipleEditorTabs';
+
+	constructor() {
+		super({
+			id: ShowMultipleEditorTabsAction.ID,
+			title: {
+				value: localize('showMultipleEditorTabs', "Show Multiple Editor Tabs"),
+				original: 'Show Multiple Editor Tabs'
+			},
+			category: Categories.View,
+			precondition: ContextKeyExpr.equals('config.workbench.editor.showTabs', 'multiple').negate(),
+			f1: true
+		});
+	}
 
-		return configurationService.updateValue(this.settingId, newSettingValue);
+	override run(accessor: ServicesAccessor): Promise<void> {
+		const configurationService = accessor.get(IConfigurationService);
+		return configurationService.updateValue('workbench.editor.showTabs', 'multiple');
 	}
 }
+registerAction2(ShowMultipleEditorTabsAction);
 
-// --- Toggle Tabs Visibility
+// --- Show Single Editor Tab
 
-export class ToggleTabsVisibilityAction extends BaseToggleBooleanSettingAction {
+export class ShowSingleEditorTabAction extends Action2 {
 
-	static readonly ID = 'workbench.action.toggleTabsVisibility';
+	static readonly ID = 'workbench.action.showEditorTab';
```

Comment:
People might have keybindings or muscle memory built up for this command, should we have it as a way to toggle between `multi` and `single` still?

Reply:
Do you really think people are consistently switching between single and multiple tabs and have a keybinding for that?